src/app/zones/routes.ts

```diff
@@ -7,165 +10,72 @@ export const actions = (): RouteRecordRaw[] => {
               path: '',
-              name: 'zone-egress-tabs-view',
-              meta: {
-                module: 'zone-egresses',
-              },
-              component: () => import('@/app/zones/views/ZoneTabsView.vue'),
-              children: [
-                {
-                  path: '',
-                  name: 'zone-egress-list-view',
-                  component: () => import('@/app/zones/views/ZoneEgressListView.vue'),
-                },
-              ],
+              name: 'zone-cp-list-view',
+              component: () => import('@/app/zones/views/IndexView.vue'),
             },
             {
-              path: ':zoneEgress',
-              name: 'zone-egress-detail-tabs-view',
-              component: () => import('@/app/zones/views/ZoneEgressDetailTabsView.vue'),
+              path: ':zone',
+              name: 'zone-cp-detail-abstract-view',
               children: [
                 {
                   path: '',
-                  name: 'zone-egress-detail-view',
-                  component: () => import('@/app/zones/views/ZoneEgressDetailView.vue'),
-                },
-                {
-                  path: 'xds-config',
-                  name: 'zone-egress-xds-config-view',
-                  component: () => import('@/app/zones/views/ZoneEgressXdsConfigView.vue'),
-                },
-                {
-                  path: 'stats',
-                  name: 'zone-egress-stats-view',
-                  component: () => import('@/app/zones/views/ZoneEgressStatsView.vue'),
-                },
-                {
-                  path: 'clusters',
-                  name: 'zone-egress-clusters-view',
-                  component: () => import('@/app/zones/views/ZoneEgressClustersView.vue'),
-                },
-                {
-                  path: 'config',
-                  name: 'zone-egress-config-view',
-                  component: () => import('@/app/zones/views/ZoneEgressConfigView.vue'),
+                  name: 'zone-cp-detail-tabs-view',
+                  component: () => import('@/app/zones/views/item/IndexView.vue'),
+                  redirect: { name: 'zone-cp-detail-view' },
+                  children: [
+                    {
+                      path: 'overview',
+                      name: 'zone-cp-detail-view',
+                      component: () => import('@/app/zones/views/item/DetailView.vue'),
+                    },
+                    {
+                      path: 'config',
+                      name: 'zone-cp-config-view',
+                      component: () => import('@/app/zones/views/item/ConfigView.vue'),
+                    },
+                    ...ingresses().items(),
+                    ...egresses().items(),
+                  ],
                 },
+                ...ingresses().item(),
+                ...egresses().item(),
               ],
             },
           ],
         },
-      ],
-    },
+
+      ]
+      : []),
```

Comment:
The entire green above here in this file is a little weird as this is were we use `can('use zones')` to give us different configurations of routing depending on whether zones are enabled or not. Probably later at some point, we'll probably do this higher up in our wiring root. I just did it here for now.